BlinkID/example/lib/document_scanner.dart

```diff
@@ -0,0 +1,173 @@
+                    recognizerCollection: RecognizerCollection(<Recognizer>[
+                      BlinkIdMultiSideRecognizer()
+                        ..recognitionModeFilter = (RecognitionModeFilter()..enablePhotoId = false)
+                        ..skipUnsupportedBack = true
+                        ..anonymizationMode = AnonymizationMode.None,
+                    ]),
+                    onResult: (result) {
+                      if (_result != null) return Future.value(false);
+                      _result = result;
+
+                      Navigator.of(context).pop(_result);
+                      return Future.value(false);
+                    },
+                    onFirstSideScanned: () => _firstSideScannedNotifier.value = true,
+                    onDetectionStatusUpdate: (status) => _detectionStatusNotifier.value = status,
+                    useFrontCamera: false,
+                    flipFrontCamera: false),
+              ),
+              Positioned(
+                  left: 40,
+                  right: 40,
+                  top: 40,
+                  child: Column(
+                    children: [
+                      ValueListenableBuilder(
+                          valueListenable: _firstSideScannedNotifier,
+                          builder: (_, firstSideScanned, __) => Text(
+                                firstSideScanned ? 'Scan the back of the document.' : 'Scan the front of the document.',
+                                style: Theme.of(context)
+                                    .textTheme
+                                    .headlineLarge
+                                    ?.copyWith(color: Colors.white, shadows: _shadows),
+                              )),
+                      const SizedBox(height: 20),
+                      ValueListenableBuilder(
+                          valueListenable: _detectionStatusNotifier,
+                          builder: (_, detectionStatus, __) => Text(
+                                detectionStatus?.directions ?? '',
+                                style: Theme.of(context)
+                                    .textTheme
+                                    .headlineMedium
+                                    ?.copyWith(color: Colors.white, shadows: _shadows),
+                              )),
+                    ],
+                  )),
+              Align(
+                  alignment: Alignment.topLeft,
+                  child: IconButton(
+                      padding: const EdgeInsets.all(40),
+                      icon: const Icon(
+                        Icons.close,
+                        color: Colors.white,
+                        shadows: _shadows,
+                        size: 28,
+                      ),
+                      onPressed: () => Navigator.of(context).pop(null))),
+            ],
+          ),
+        ),
+      );
+}
+
+class _DocumentScanner extends StatelessWidget {
+  const _DocumentScanner({
```

Comment:
What is the point of this Widget, if it mostly relays the parameters to `MicroblinkScannerWidget`?